packages/gatsby-theme-styleguide/src/gatsby-components/Menu/MenuItem.js

```diff
@@ -57,24 +59,42 @@ const MenuLink = styled(Link)`
     `}
 `
 
-const MenuItem = ({ to, content, level, isPartOfCurrentlocation }) =>
-  to ? (
-    <MenuLink
-      to={to}
-      level={level}
-      isPartOfCurrentlocation={isPartOfCurrentlocation}
-    >
-      {content}
-    </MenuLink>
-  ) : (
-    <p>{content}</p>
-  )
+class MenuItem extends Component {
+  constructor(props) {
+    super(props)
```

Comment:
This will allow you to access the `this` element in the `handleClick` method

```js
this.handleClick = this.handleClick.bind(this);
```

Reply:
fixed